spring-boot/src/main/java/org/springframework/boot/context/embedded/undertow/UndertowEmbeddedServletContainerFactory.java

```diff
@@ -442,6 +482,22 @@ public void setDirectBuffers(Boolean directBuffers) {
 		this.directBuffers = directBuffers;
 	}
 
+	public void setBaseDirectory(File baseDirectory) {
```

Comment:
Unlike Tomcat, Undertow doesn't have a base directory so I don't think that makes sense. I think it would be better to configure the access logs directory directly.